### PR TITLE
Audit & fix Information.md

### LINKED ISSUE #44: Audit & fix Information.md page
This is a sub-issue of #18.
I'd like to audit and fix the Information.md page based on tasks listed below.

## Check List
- [x] Look for any typos
- [x] Make sure all Markdown is correctly done. The conversion was done automatically by a tool, so there may be more idiomatic ways to do some things
- [x] Check the page in a Desktop browser. Does it look OK? Are there any issues that need to be fixed?
- [x] Check the page in a Mobile browser. Does it look OK? Are there any issues that need to be fixed?
- [x] Try the page in both Light and Dark mode. Are there any issues that need to be fixed?
- [x] Try running the page through Lighthouse: https://developers.google.com/web/tools/lighthouse. Are there any issues that need to be fixed?
- [x] Try running the page through Web Hint: https://webhint.io/. Are there any issues that need to be fixed?
- [x] Tables that are really wide (types.md) could be converted to use SVG or flex/grid. Make wide visual tables fit in the available viewport. Another example is the table in docs/D-Modularity/pointers.md, where we should reduce the vertical padding/margins on tables. The tables are too large, and probably shouldn't be tables at all.
- [x] Prefer Markdown tables vs. raw HTML tags
- [x] Fix colours in tables so it uses global styles vs. inline styles
- [x] Use alt tags for all images. See https://supercooldesign.co.uk/blog/how-to-write-good-alt-text for suggestions
- [x] Avoid blockquote for indentation, and prefer q or blockquote only when actually quoting. Use Admonitions instead. Example: docs/D-Modulatirty/pointers.md: there are places where > Blockquote are being used and Admonitions could replace it. “Note:” Lightbulb info Admonition.
- [x] Make sure images work in dark and light modes. For example, docs/8-Computations/logic.md Flags image is white background, needs to work in dark. Create a “negative” or SVG so we can do Dark Mode? Can we programmatically create this negative?
- [x] Change colours for tables (don’t use Yellow). We should use the default colour theme.
- [x] Avoid using &lt;b&gt; and &lt;i&gt; to do emphasis.
- [x] Use backticks and all code for memory addresses and such in tables that use things like void, int, function names, memory addresses.
- [x] Make sure all code blocks use syntax highlighting
- [x] Centre images horizontally (esp. when they are small) so they work well at all responsive sizes
- [x] Check sub-script and super-script that they are correctly used in all cases
- [x] Consider including inter-site links to other pages that have related info
- [x] All links should have descriptive text for accessibility
- [x] Make sure that sub-headings (right-hand side) for each document are correct.
- [x] Improve readability of code comments (i.e., // and /*...*/) in dark mode (even light mode might need some work)
- [x] In places where ***...*** was used as a fourth heading level, use #### instead.
- [x] Fix Frontmatter for page to include proper id, title, slug, etc.

### PR DESCRIPTION
**Pull request by Yoda-Canada**
Fixes #44
Steps done to modify and check Information.md:

1. Fixed images work in dark modes. The memory model image's downward arrows and the letter in the bottom right corner are not clear in dark modes. I save the image in the image folder and link it to the information page. The issue is fixed.
2. Added alt text for diagram images.
3. The images can't display in all models. Change all the image path from "/img/*.jpg" to "../../static/img/*.jpg". The issue is fixed.
4. Fix Frontmatter for the page to include id, title, slug, and description.
5. Replace blockquote with Admonitions for clarity.
image
image

### CODE REVIEW

**Comment by AndreWillomitzer** [comment]
@Yoda-Canada A few things that may be causing the deployment to fail and I noticed:

1.  To fix the dark mode image issue, I would suggest wrapping them in a `<span>` tag and adding class "mdImage" to it. Then, when your change gets merged it will have the global CSS applied that I did in my merged PR: 
```
.mdImg > p > img{
  background-color: #ffffff;
}
```

2.  I am not actually sure you needed to add `` and `<div>` tags around the images. For my fix all I did was add the alt description to the Markdown image itself and then wrap it in a `<span>`. The difference between div and span are that divs are block elements, and span are inline. By using span, you won't mess with any mobile/screen resizing by accident.

3.  For regular text, I don't believe it is necessary to wrap them in `<p>` tags because we're using a Markdown file and so regular text that isn't preceeded by a symbol like `###` is treated like normal text.

Otherwise, great job looks like you put tons of effort in to the changes 🚀 !

File: package.json [hunk 1]
```diff
@@ -42,8 +42,12 @@
...
   },
   "devDependencies": {
+<<<<<<< HEAD
     "execa": "^5.1.1",
     "mr-pdf": "^1.0.7",
     "wait-on": "^6.0.0"
+=======
+    "docusaurus": "^1.14.7"
+>>>>>>> issue-44
   }
```
Comment (suhhee1011) [inline package.json:52]: I think you forgot to erase the `<<<<<<<<HEAD`, `======`, `>>>>>>issue-44`tag when you merge it to the main branch on your repository.

Comment (AndreWillomitzer) [reply package.json:52]: Good point that could be why deployment failed. Nice spot :D

File: docs/A-Introduction/information.md [hunk 2]
```diff
@@ -30,9 +34,12 @@ Since bits are too numerous to handle individually, modern computers transfer an
 
 The fundamental addressable unit of RAM is the byte.  One byte consists of 2 nibbles.  Each nibble consists of 4 bits.
 
-![](/img/bytenibbit.jpg)
+<div  align="left">
+<img src="../../static/img/bytenibbit.jpg" alt="A bytes tree image" />
```
Comment (dhillonks) [inline docs/A-Introduction/information.md:38]: The `alt` text could be more descriptive. For instance: `Tree showing relative sizes of bit, nibble and byte`

File: docs/A-Introduction/information.md [hunk 3]
```diff
@@ -78,7 +85,9 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
 
 The memory model for organizing information stored in RAM is linear.  Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
 
-![RAM memory model: linearly mapped \(continuous sequence\)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
+<div  align="left">
```
Comment (dhillonks) [inline docs/A-Introduction/information.md:88]: **center** the images and remove the extra whitespace in the tag
```suggestion
<div align="center">
```

**Review by dhillonks — CHANGES_REQUESTED** [review]

**Comment by oliver-pham** [comment]
I think you should remove `hint-report/https-webhint-io.html` and `yarn.lock` from your commit. These files are generated locally and not relevant to your changes.

File: docs/A-Introduction/information.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 ---
+id: information
 sidebar_position: 2
+title: Information
+slug: /A-Introduction/information
```
Comment (GMOTGIT) [inline docs/A-Introduction/information.md:5]: Hey, great work on this Md file,
Something that I noticed was:

Slug should be standardized in small caps and without the "A-"
Check https://github.com/Seneca-ICTOER/IPC144/issues/64 for more details.

Thank you.

**Review by GMOTGIT — CHANGES_REQUESTED** [review]

File: docs/A-Introduction/information.md (hunk 4 above)
Comment (Yoda-Canada) [reply docs/A-Introduction/information.md:5]: thx, I have modified.

**Comment by Yoda-Canada** [comment]
> @Yoda-Canada A few things that may be causing the deployment to fail and I noticed:
> 
> 1. To fix the dark mode image issue, I would suggest wrapping them in a `<span>` tag and adding class "mdImg" to it. Then, when your change gets merged it will have the global CSS applied that I did in my merged PR:
> 
> ```
> .mdImg > p > img{
>   background-color: #ffffff;
> }
> ```
> 
> 1. I am not actually sure you needed to add `` and `<div>` tags around the images. For my fix all I did was add the alt description to the Markdown image itself and then wrap it in a `<span>`. The difference between div and span are that divs are block elements, and span are inline. By using span, you won't mess with any mobile/screen resizing by accident.
> 2. For regular text, I don't believe it is necessary to wrap them in `<p>` tags because we're using a Markdown file and so regular text that isn't preceeded by a symbol like `###` is treated like normal text.
> 
> Otherwise, great job looks like you put tons of effort in to the changes 🚀 !

@AndreWillomitzer Thank you, I have modified all you mentioned.

**Comment by Yoda-Canada** [comment]
> I think you should remove `hint-report/https-webhint-io.html` and `yarn.lock` from your commit. These files are generated locally and not relevant to your changes.

@oliver-pham I have added these files in the .gitignore, thank you!

**Comment by Yoda-Canada** [comment]
> I think you should remove `hint-report/https-webhint-io.html` and `yarn.lock` from your commit. These files are generated locally and not relevant to your changes.

@oliver-pham yarn.lock shouldn't remove.

**Review by Yoda-Canada — COMMENTED** [review]
thx, I have modified.

File: docs/A-Introduction/information.md [hunk 5]
```diff
@@ -90,10 +103,17 @@ For example, 4 Gigabytes of RAM
...
+</span>
+
+:::note 
 
->**Note** that each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
+Each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
 
+:::
 
 ### Sets of Bytes
```
Comment (mqnguyen5) [inline docs/A-Introduction/information.md:119]: I would suggest escaping any `*` character that represents multiplication (line 122-128), as some of them will get converted into `_` after being formatted with Prettier. See #56 for more info.

**Review by mqnguyen5 — CHANGES_REQUESTED** [review]

File: docs/A-Introduction/information.md (hunk 5 above)
Comment (TueNguyen2911) [reply docs/A-Introduction/information.md:119]: > I would suggest escaping any `*` character that represents multiplication (line 122-128), as some of them will get converted into `_` after being formatted with Prettier. See #56 for more info.

Did you mean the multiplication from line 115-122?

Comment (Yoda-Canada) [reply docs/A-Introduction/information.md:119]: I have modified them, thx

**Comment by Yoda-Canada** [comment]
Since the commits can't be squashed to one, I close this PR and open a new PR #101. Thanks to all contributors.

**Pull request closed without merging**